src/sql/platform/connection/common/constants.ts

```diff
@@ -65,3 +65,8 @@ export const UNSAVED_GROUP_ID = 'unsaved';
 /* Server Type Constants */
 export const sqlDataWarehouse = 'Azure SQL Data Warehouse';
 export const gen3Version = 12;
+
+/* SQL Server Password Reset Constants */
+export const sqlPasswordErrorCode = 18488;
+export const sqlPasswordChangeErrorHeader = 'Failure when attempting to change password'
+export const sqlPasswordMismatchMessage = 'Passwords entered do not match\n\nPress OK and enter the exact same password in both boxes.';
```

Comment:
1. we shouldn't have SQL Server specific stuff in core
2. shouldn't use English strings directly here.